tests/coprocessor/test_select.rs

```diff
@@ -316,6 +316,36 @@ fn test_aggr_count() {
 }
 
 #[test]
+fn test_aggr_first() {
+    let count = 10;
+    let (mut store, mut end_point, ti) = initial_data(count);
```

Comment:
Could you add some comment for the data in the table?
You can add it here or at the definition of initial_data().